sources/engine/Xenko.Input/Simulated/PointerSimulated.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Xenko contributors (https://xenko.com) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
```

Comment:
New file headers should not contain Silicon Studio Corp copyright, only xenko part